### PR TITLE
For CASSANDRA-10492, fix test bug in paging_test.py where assert_read…

### PR DESCRIPTION
**Pull request by aweisberg**
…_timeout_or_failure doesn't handle OperationTimedOut

### CODE REVIEW

**Comment by mambocab** [comment]
If I read the driver docs correctly, `OperationTimedOut`s happen on the driver side, while `ReadTimeout`s propagate C* errors. If we want to treat those the same, I'm +1 on the behavioral change.

For the exceptions we `pass` on in that function, could you please move them all to a single `except` block like so:

```python
    except (ReadTimeout, ReadFailure, OperationTimedOut):
        pass
```

Might as well get some style cleanup in while we're churning. Thanks!

**Comment by aweisberg** [comment]
Well now I'm not so sure. The timeout for the read operation is 1 second, but the default timeout in the driver for OperationTimedOut is 10 seconds. Doesn't seem like it should every throw OperationTimedOut.

**Comment by aweisberg** [comment]
Turns out the issue was that the timeout server side was not as small as the test thought so it was hitting the timeout in the driver. I updated the server side configuration to the expected 1 second timeout and now it throws the expected exceptions.

File: upgrade_tests/paging_test.py [hunk 1]
```diff
@@ -24,7 +24,6 @@ def assert_read_timeout_or_failure(session, query):
     except ReadFailure as e:
         pass
 
-
```
Comment (mambocab) [inline upgrade_tests/paging_test.py]: Can you add this line back, please?

**Comment by mambocab** [comment]
Ran them locally, looks good. Can you add back that whitespace and ping me for merge?

**Comment by mambocab** [comment]
Thanks, merging.

**Pull request merged**